content/building-blocks/store-and-document-your-data/store-data/aws-s3-buckets.md

```diff
@@ -33,15 +37,20 @@ Learn more about AWS credentials [here](https://docs.aws.amazon.com/general/late
 
 ## Code

```

Comment:
Better subtitle: "Set up a connection with AWS S3 using Python" or better instead of "Code"?

Reply:
Good point, indeed the `## Code` header may sound a bit generic. I have substituted it for "Handle AWS S3 through Python" as I think that "Set up a connection with AWS S3 using Python" may enter in conflict with the sub-header immediately below which refers to the the process of establishing a connection with S3 (Which I have renamed as you proposed).

I feel that after these changes are adapt better to the content! 🎯